Gemfile

```diff
@@ -2,12 +2,6 @@ source 'https://rubygems.org'
 
 gem 'kafo', '~> 4.1'
 gem 'librarian-puppet'
-gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '~> 6.0'
-if RUBY_VERSION < '2.1'
-  gem 'puppet-strings', '< 2.0.0'
-else
-  gem 'puppet-strings'
-end
 gem 'rake'
 gem 'rdoc', '< 6' if RUBY_VERSION < '2.2'
```

Comment:
Unrelated, but we can drop this since we dropped Ruby 2. Same for `semverse` but I guess that can go in a different PR as well.